### PR TITLE
PWN-7343 - Provide mint address for Erc20Token

### PR DESCRIPTION
**Pull request by Davran331122**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7343

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]

**Review by NikolaiEmelianov — APPROVED** [review]

File: ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/ERC20Token.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package org.p2p.ethereumkit.external.model
+
+import org.p2p.ethereumkit.internal.models.EthAddress
+
+enum class ERC20Token(val contractAddress: String, val mintAddress: String) {
```
Comment (gslevinkov) [inline ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/ERC20Token.kt:5]: Erc20Tokens

File: ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/ERC20Token.kt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        contractAddress = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32",
+        mintAddress = "HZRCwxP2Vq9PCpPXooayhJ2bxTpo5xfpQrwB1svh332p"
+    ),
+    HXRO(
+        contractAddress = "0x4bD70556ae3F8a6eC6C4080A0C327B24325438f3",
+        mintAddress = "HxhWkVpk5NS4Ltg5nij2G671CKXFRKPK8vy271Ub4uEK"
+    );
+
+    companion object {
+        fun findItem(contractAddress: EthAddress): ERC20Token {
```
Comment (gslevinkov) [inline ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/ERC20Token.kt:68]: findToken

File: ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/EthTokenMetadata.kt [hunk 3]
```diff
@@ -8,6 +8,7 @@ import java.math.BigInteger
 
 data class EthTokenMetadata(
     val contractAddress: EthAddress,
+    val mintAddress: String,
```
Comment (gslevinkov) [inline ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/EthTokenMetadata.kt:11]: is Mint Address a base58String?

File: ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/EthTokenMetadata.kt [hunk 4]
```diff
@@ -18,10 +19,12 @@ data class EthTokenMetadata(
 
 internal fun mapToTokenMetadata(
     balanceResponse: TokenBalanceResponse,
-    metadata: TokenMetadataResponse
+    metadata: TokenMetadataResponse,
+    mintAddress: String
 ): EthTokenMetadata {
-    return EthTokenMetadata(
+    return  EthTokenMetadata(
```
Comment (gslevinkov) [inline ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/EthTokenMetadata.kt:25]:
```suggestion
    return EthTokenMetadata(
```

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**